Dockerfile

```diff
@@ -16,6 +16,8 @@ ENV S3_PUBLISHED_LOCATIONS_IPS_BUCKET 'StubBucket'
 ENV S3_PUBLISHED_LOCATIONS_IPS_OBJECT_KEY 'StubKey'
 ENV S3_SIGNUP_WHITELIST_BUCKET 'StubBucket'
 ENV S3_SIGNUP_WHITELIST_OBJECT_KEY 'StubKEY'
+ENV S3_ORGANISATION_NAMES_BUCKET 'StubBucket'
```

Comment:
I assume the bucket name will actually be something like `S3_PRODUCT_PAGE_DATA` and the bucket key will then be `S3_ORGANISATION_NAMES_OBJECT_KEY` `S3_EMAIL DOMAINS_OBJECT_KEY`?